lib/board.rb

```diff
@@ -63,10 +63,10 @@ def place(ship, set_of_coordinates)
   def render(secretboard = false)
 
     "  1 2 3 4 \n" +
-    "A #{cells["A1"].render(secretboard)} #{cells["A2"].render(secretboard)} #{cells["A3"].render(secretboard)} #{cells["A4"].render(secretboard)}\n" +
-    "B #{cells["B1"].render(secretboard)} #{cells["B2"].render(secretboard)} #{cells["B3"].render(secretboard)} #{cells["B4"].render(secretboard)}\n" +
-    "C #{cells["C1"].render(secretboard)} #{cells["C2"].render(secretboard)} #{cells["C3"].render(secretboard)} #{cells["C4"].render(secretboard)}\n" +
-    "D #{cells["D1"].render(secretboard)} #{cells["D2"].render(secretboard)} #{cells["D3"].render(secretboard)} #{cells["D4"].render(secretboard)}"
+    "A #{cells["A1"].render(secretboard)} #{cells["A2"].render(secretboard)} #{cells["A3"].render(secretboard)} #{cells["A4"].render(secretboard)} \n" +
+    "B #{cells["B1"].render(secretboard)} #{cells["B2"].render(secretboard)} #{cells["B3"].render(secretboard)} #{cells["B4"].render(secretboard)} \n" +
+    "C #{cells["C1"].render(secretboard)} #{cells["C2"].render(secretboard)} #{cells["C3"].render(secretboard)} #{cells["C4"].render(secretboard)} \n" +
+    "D #{cells["D1"].render(secretboard)} #{cells["D2"].render(secretboard)} #{cells["D3"].render(secretboard)} #{cells["D4"].render(secretboard)} \n"
```

Comment:
It is AWESOME that you looked back at previous versions to find the solution! Proud of you and thank you for your hard work.